### ISSUE TITLE
Simplifying manual identity matching in social smells notebook

### ISSUE THREAD

**Issue by tuejari** [sailuh/kaiaulu#138]
In the social smells notebook, it is strongly encouraged to manually verify the identity matching results. 
To help me with that process, I wrote the following lines of code:

````{r}
```{r eval = FALSE}
# Select id and name from project_git
authors_git <- project_git[,c("author_name_email", "identity_id"), drop=FALSE]
authors_reply <- project_reply[,c("reply_from", "identity_id"), drop=FALSE]

# Select all unique rows to get all unique authors
authors_git <- unique(authors_git)
authors_reply <- unique(authors_reply)

# Transform all names to lowercase to fix sorting (otherwise we get A B C a b c)
authors_git$author_name_email= tolower(authors_git$author_name_email)
authors_reply$reply_from= tolower(authors_reply$reply_from)

# Order the rows alphabetically by name
authors_git <- authors_git[order(author_name_email)]
authors_reply <- authors_reply[order(reply_from)]

# View the authors for inspection of the identity matching. 
# NOTE: these are not the tables to edit. Instead, you should edit the
# project_git and project_reply
View(authors_git)
View(authors_reply)

# View the projects for inspection of which rows/cells to adjust
View(project_git)
View(project_reply)
```
````

Basically, we copy `project_git` and `project_reply` and then select unique rows in the copies (note that in `project_git` and `project_reply` the same identity can occur in multiple rows). We then sort the names (helps in identifying some cases, similar to the example in https://github.com/sailuh/kaiaulu/issues/136#issuecomment-933772541). Then we use `View` so we can inspect the tables. Note that I added `{r eval = FALSE}` at the top, because to my understanding not everyone manually inspects the identity matching.  

It might be worth considering adding something similar to the above after the following lines in the notebook.

https://github.com/sailuh/kaiaulu/blob/44cebf8409d1403158a6f7e11f76c16d7662b1c3/vignettes/social_smell_showcase.Rmd#L211-L233
